<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>627</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>B Yekan</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ربات آسان ساز ثنا</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>1101</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Yekan</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::TabShape::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="sendToDivanTab">
    <property name="font">
     <font>
      <family>B Yekan</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>ارسال لوایح به دیوان عدالت اداری</string>
    </attribute>
    <widget class="QPushButton" name="newPostBtn">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Yekan</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ارسال جدید </string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="postsBox">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>90</y>
       <width>1051</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Yekan</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>ارسال ها</string>
     </property>
     <widget class="QTableWidget" name="postsTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1021</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Yekan</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>عنوان</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>تاریخ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>تعداد لایحه ها</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>وضعیت</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>زمان شروع</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>زمان اتمام</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>تعداد موفق</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>تعداد خطا</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>عملیات</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Yekan</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>جست و جو </string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fetchEblagh">
    <attribute name="title">
     <string>دریافت ابلاغیه</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
